### DISCUSSION TITLE
apk incompatible with android 14 one ui 6.0

### DISCUSSION THREAD

**Discussion by Ranyxhave** [tribalfs/GalaxyMaxHzPub#195] [Q&A] [answered]
Hi, I'm Brazilian, I couldn't download it, it says that the apk is incompatible with the version of my cell phone, which is an S21 fe, update the application or help me please

**Comment by tribalfs** [comment, marked as answer]
It needs to be installed using ADB. Read and follow the [instructions](https://github.com/tribalfs/GalaxyMaxHzPub/releases/tag/v8.1) provided
